common/cah-model.cpp

```diff
@@ -73,16 +73,16 @@ bool cah_model::prompt_trigger_camera_accuracy_health(device_model & dev_model,
                     sd->s->set_option(RS2_OPTION_TRIGGER_CAMERA_ACCURACY_HEALTH, 1.0f);
                     device_calibration dev_cal(dev_model.dev);
                     // Register AC status change callback
-                    if (!registered_to_callback)
+                    if (!_registered_to_callback)
                     {
-                        registered_to_callback = true;
-                        cah_state = model_state_type::PROCESS_MODAL;
+                        _registered_to_callback = true;
+                        _state = model_state_type::PROCESS_MODAL;
                         // We switch to process state without a guarantee that the process really started,
                         // Set a timeout to make sure if it is not started we will allow closing the window.
-                        cah_process_timeout.start(std::chrono::seconds(30)); 
```

Comment:
This, too...

Reply:
The code is outdated, it has been fixed already.